### PR TITLE
Forward port 3 to 6, Jan 29 2023

### PR DESCRIPTION
**Pull request by nkoenig**
# ➡️ Forward port

Port ign-rendering3 to ign-rendering6

Branch comparison: [https://github.com/gazebosim/<REPO>/compare/<TO_BRANCH>...<FROM_BRANCH>](https://github.com/gazebosim/gz-rendering/compare/ign-rendering6...ign-rendering3)

**Note to maintainers**: Remember to **Merge** with commit (not squash-merge or rebase)

### CODE REVIEW

**Comment by iche033** [comment]
Did a first pass review. Started fixing warnings and tests in https://github.com/gazebosim/gz-rendering/pull/819

**Review by ahcorde — CHANGES_REQUESTED** [review]
conflicts

File: Changelog.md [hunk 1]
```diff
@@ -740,7 +740,7 @@
 
 1. Remove Windows Warnings
     * [Pull request #206](https://github.com/ignitionrobotics/ign-rendering/pull/206)
-    * [Pull request #213](https://github.com/ignitionrobotics/ign-rendering/pull/213)
+    * [Pull request #213](https://github.cohttps://github.com/gazebosim/<REPO>/compare/<TO_BRANCH>...<FROM_BRANCH>m/ignitionrobotics/ign-rendering/pull/213)
```
Comment (iche033) [inline Changelog.md:743]: revert this change?

Comment (nkoenig) [reply Changelog.md]: Done

**Review by iche033 — APPROVED** [review]
looks good to me. Demos are working fine

**Review by ahcorde — APPROVED** [review]

**Pull request merged**